crates/beautifier/src/lib.rs

```diff
@@ -0,0 +1,264 @@
+/*
+ * Copyright 2022 Fluence Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#![deny(
+    dead_code,
+    nonstandard_style,
+    unused_imports,
+    unused_mut,
+    unused_variables,
+    unused_unsafe,
+    unreachable_patterns
+)]
+
+use air_parser::ast;
```

Comment:
I suggest to move all logic from `lib.rs` to a separate `beautifier.rs` file and add here only necessary reexports + two more functions
`beautify(air_script: &str, &mut impl io::Write) -> IoResult<()>`,
`beautify_to_string` - it'll acts exactly as one in tests, and in the future we could export it in the Wasm side

Reply:
Will do as the last refactoring.